homeassistant/components/recorder/purge.py

```diff
@@ -36,41 +28,39 @@ def purge_old_data(
 
     Cleans up an timeframe of an hour, based on the oldest record.
     """
+    return try_database_job(
```

Comment:
Maybe make this a decorator?

Reply:
Fixed in 9c3e78adc20a473775a207f3fadce9d7caa8bb61